sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/test/scala/com/azure/cosmos/spark/CosmosPartitionPlannerITest.scala

```diff
@@ -185,6 +195,61 @@ class CosmosPartitionPlannerITest
       Some(90))
   }
 
+  "createInitialOffset" should "only ever retrieve one page of changes per feedRange" in {
+
+    val client = CosmosClientCache.apply(clientConfig, None)
```

Comment:
why can't we use the existing client from the CosmosClient trait?

Reply:
Fixed